openmc/model/funcs.py

```diff
@@ -345,6 +345,54 @@ def get_hexagonal_prism(edge_length=1., orientation='y', origin=(0., 0.),
     return prism
 
 
+def cylinder_from_points(p1, p2, r, **kwargs):
+    """Return cylinder defined by two points passing through its center.
+
+    Parameters
+    ----------
+    p1, p2 : 3-tuples
+        Coordinates of two points that pass through the center of the cylinder
+    r : float
+        Radius of the cylinder
+    kwargs : dict
+        Keyword arguments passed to the :class:`openmc.Quadric` constructor
+
+    Returns
+    -------
+    openmc.Quadric
+        Quadric surface representing the cylinder.
+
+    """
+    # Get x, y, z coordinates of two points
+    x1, y1, z1 = p1
+    x2, y2, z2 = p2
+
+    # Define intermediate terms
+    dx = x2 - x1
+    dy = y2 - y1
+    dz = z2 - z1
+    cx = y1*z2 + y2*z1
```

Comment:
These yield PEP8 E226: whitespace around operators.  Also, I really do like the whitespace.  Could we add that in?

Reply:
Sometimes linters are not very smart. If you go read the [relevant section of PEP8](https://www.python.org/dev/peps/pep-0008/#other-recommendations), you'll see that the style I have is what is recommended. They only recommend whitespace around the lowest-precedence operators, not all.

Reply:
ah alright. darn.